plugin/pkg/scheduler/algorithm/scheduler_interface_test.go

```diff
@@ -30,9 +30,11 @@ type schedulerTester struct {
 	nodeLister NodeLister
 }
 
+var EnableGeneralPredicates = true
+
 // Call if you know exactly where pod should get scheduled.
 func (st *schedulerTester) expectSchedule(pod *api.Pod, expected string) {
-	actual, err := st.scheduler.Schedule(pod, st.nodeLister)
+	actual, err := st.scheduler.Schedule(pod, st.nodeLister, !EnableGeneralPredicates)
```

Comment:
The stuff defined in this file doesn't appear to be used anywhere. Am I missing something? Should we just delete the file?